### DISCUSSION TITLE
连接界面速度全部显示NaN

### DISCUSSION THREAD

**Discussion by rickblue** [libnyanpasu/clash-nyanpasu#671] [General]
Nyanpasu版本：1.5.1-alpha-0b4d3q7
clash meta：alpha-288899a
同样配置另一台电脑没问题，webui也没问题，研究半天也没搞定，希望大佬给指点下。
![屏幕截图 2024-03-25 093339](https://github.com/LibNyanpasu/clash-nyanpasu/assets/14907722/093acb61-a70f-406d-947e-9457a4c87dd6)

**Comment by greenhat616** [comment]
已经修复

**Reply by rickblue** [reply]
好了，大佬辛苦！
